xbmc/addons/kodi-addon-dev-kit/include/kodi/gui/ListItem.h

```diff
@@ -213,12 +218,13 @@ namespace gui
     ///
     /// @param image                    The image to use for.
     ///
-    /// @note Alternative can be \ref SetArt used
+    /// @deprecated Use \ref SetArt instead
     ///
     ///
     void SetIconImage(const std::string& image)
```

Comment:
This PR breaks `kodi.binary.global.gui` compatibility, which means at least a rebuild is required.
Because of that you could just remove this function.